.github/scripts/compute_and_upload_ci_wait_time_metric.py

```diff
@@ -0,0 +1,389 @@
+import os
```

Comment:
This script looks complicated but well modularize.  Should we add some unit tests for it to make sure that these functions are well behave.  The input could be a mock JSON from Rockset

Reply:
Hmm...let me think about what kind of tests might be appropriate here.

I'm thinking that testing this all in one go (i.e. "Given this sample Rockset output, what's the final stats you'd generate") is not a helpful test, since it would break whenever anyone makes any change to this query.

Each method would need it's own unit test for the testing to be useful